doc/nrf/libraries/bluetooth_services/services/fast_pair.rst

```diff
@@ -20,19 +20,27 @@ Characteristics
 The Fast Pair GATT characteristics are described in detail in the `Fast Pair GATT Characteristics`_ documentation.
 The implementation in the |NCS| follows these requirements.
 
+The Fast Pair service also contains additional GATT characteristics under the following conditions:
+
+* The Additional Data GATT characteristic is enabled when an extension requires it.
+  Currently, only the Personalized Name extension (:kconfig:option:`CONFIG_BT_FAST_PAIR_EXT_PN`) requires this characteristic.
```

Comment:
```suggestion
The Fast Pair service also contains additional Data GATT characteristic that is enabled when an extension requires it.
Currently, only the Personalized Name extension (:kconfig:option:`CONFIG_BT_FAST_PAIR_EXT_PN`) requires this characteristic.
```

Reply:
Discussed offline. We expect to have one more entry here soon